### DISCUSSION TITLE
〔 Comments Section 〕Fallout Shelter Online: Base and Resources

### DISCUSSION THREAD

**Discussion by morgosus** [morgosus/morgosus.github.io#11] [Comments]
A comments section for the [Fallout Shelter Online: Base and Resources](https://www.toms.click/gaming/foso/2021/08/13/fallout-shelter-base-and-resources.html) post.

**Comment by Emptycz** [comment]
What resource upgrade sequence do you recommend?

**Reply by morgosus** [reply]
I'd go with power → water → food → caps, as caps are plentiful in the early game and power is not, water can be used to upgrade your production through N and R perks. In the endgame, when your overseer's office and mystery treasure are somewhat done, you'll want to switch it up to prioritizing water → food → power → caps, as further upgrades can wait, but upgrading the dwellers can not. It'll also be very resource costly.
